README.md

```diff
@@ -22,7 +22,8 @@ docker-machine create --driver amazonec2 kvstore
 ```
 eval $(docker-machine env kvstore)
 export KV_IP=$(docker-machine ip kvstore)
-docker run -d -p 8500:8500 -h consul --restart=always progrium/consul -server -bootstrap
+docker-machine ssh kvstore
```

Comment:
That should not have had happened if you did:

```
 eval $(docker-machine env kvstore)
 export KV_IP=$(docker-machine ip kvstore)		
```

The first command will setup all of the required environment variables so that when you do `docker run ... ` it exectues on the AWS instance. You can double check that by running `docker info` and also by running `echo $KV_IP`. The later should give you a well known AWS IP.